### PR TITLE
Fix refactor little

### PR DESCRIPTION
**Pull request by heavenshell**
Hi,

While I read `registration/models.py` code, I found some refactor point.
Could you reveiw this?

Also add following changes

- Updaet requirements for failing latest pytest
- Add Django 3.1 to tox.ini
- Update setup.py's classifiers

Thank you.

### CODE REVIEW

File: registration/models.py [hunk 1]
```diff
@@ -161,10 +161,7 @@ def activate_user(self, activation_key, site, get_profile=False):
                 # again. If the User is active, return the User. Else,
                 # return False as the User has been deactivated by a site
                 # administrator.
-                if profile.user.is_active:
-                    return (profile.user, False)
-                else:
-                    return (profile.user, False)
```
Comment (heavenshell) [inline registration/models.py:167]: Always return `(profile.user, False)` . So no need check `is_active`, IMO

File: registration/models.py [hunk 2]
```diff
@@ -296,7 +293,7 @@ def delete_expired_users(self):
                     profile.delete()
                     user.delete()
             except UserModel().DoesNotExist:
-                logger.warning('Deleting expired Registration profile'.format(profile))
+                logger.warning('Deleting expired Registration profile {}'.format(profile))
```
Comment (heavenshell) [inline registration/models.py:296]: Fix lint error 😉 

```
invoke lint
registration/models.py:299:32: F523 '...'.format(...) has unused arguments at position(s): 0
```

**Comment by heavenshell** [comment]
I don't know why tests are failing.
In my local env, master branch also failed.
Dose anybody know this?

File: requirements.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 -rtest-requirements.txt
 Sphinx
 twine
-pytest>3.2.1
+pytest==5.4.2
```
Comment (heavenshell) [inline requirements.txt]: pytest 6.0.0 brokes all test. so fixed at v5 latest

File: requirements.txt [hunk 4]
```diff
@@ -1,5 +1,5 @@
 -rtest-requirements.txt
 Sphinx
 twine
-pytest>3.2.1
+pytest==5.4.3
```
Comment (heavenshell) [inline requirements.txt:4]: pytest-6.0.0 breaks all tests. fix at v5 latest

File: test-requirements.txt [hunk 5]
```diff
@@ -1,6 +1,6 @@
 flake8
 coveralls
-isort
+isort==4.3.21
```
Comment (heavenshell) [inline test-requirements.txt:3]: isort v5 also fail lint

File: test-requirements.txt [hunk 6]
```diff
@@ -1,8 +1,8 @@
...
 tox-travis
 invoke==0.12.2
 docutils==0.13.1
 mock
-pytest-django==3.4.8
+pytest-django==3.9.0
```
Comment (heavenshell) [inline test-requirements.txt:8]: pytest v6.0.0 should use pytest-django 3.9.0

**Comment by joshblum** [comment]
thanks!

**Pull request merged**